CMakeLists.txt

```diff
@@ -96,6 +96,8 @@ if (CMAKE_CXX_COMPILER_ID STREQUAL "Clang")
   set (WARN_FLAGS "${WARN_FLAGS} -Wno-covered-switch-default")
   set (WARN_FLAGS "${WARN_FLAGS} -Wno-missing-noreturn -Wno-unknown-pragmas")
   set (WARN_FLAGS "${WARN_FLAGS} -Wno-gnu-zero-variadic-macro-arguments")
+  set (WARN_FLAGS "${WARN_FLAGS} -Wno-reserved-identifier")
+  set (WARN_FLAGS "${WARN_FLAGS} -Wno-suggest-destructor-override")
```

Comment:
As you see in the below, Apache ORC community maintains a separate `if` branch like `if (CMAKE_HOST_APPLE AND CMAKE_CXX_COMPILER_VERSION VERSION_GREATER "12.0")`. Could you follow that style for `clang 13.0`?

Reply:
ok I will do.

Reply:
Thanks!